doc/api/url.md

```diff
@@ -1052,7 +1068,22 @@ string serializations of the URL. These are not, however, customizable in
 any way. The `url.format(URL[, options])` method allows for basic customization
 of the output.
 
-```js
+```mjs
+import {format} from 'url';
+const myURL = new URL('https://a:b@測試?abc#foo');
+
+console.log(myURL.href);
+// Prints https://a:b@xn--g6w251d/?abc#foo
+
+console.log(myURL.toString());
+// Prints https://a:b@xn--g6w251d/?abc#foo
+
+console.log(format(myURL, { fragment: false, unicode: true, auth: false }));
+// Prints 'https://測試/?abc'
+```
+
+```cjs
+const {format} = require('url');
```

Comment:
`format` is not a very descriptive by itself (contrarily to `pathToFileURL`). That wouldn't be consistent but at least it demonstrates that both imports are valid 🙃
```suggestion
const url = require('url');
```